Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:11:30 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:29:27 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:29:28 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:29:44 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:29:45 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:29:45 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
  [1m[35m (423.0ms)[0m  [1m[35mCREATE DATABASE "app_development" ENCODING = 'unicode'[0m
  [1m[35m (283.6ms)[0m  [1m[35mCREATE DATABASE "app_test" ENCODING = 'unicode'[0m
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:36:21 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:36:22 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:36:23 +0900
  
ActiveRecord::ConnectionNotEstablished (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
):
  
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:83:in `rescue in new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:77:in `new_client'
activerecord (6.1.4.1) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `public_send'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:882:in `new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:926:in `checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:905:in `try_to_checkout_new_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:866:in `acquire_connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:588:in `checkout'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:428:in `connection'
activerecord (6.1.4.1) lib/active_record/connection_adapters/abstract/connection_pool.rb:1128:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:327:in `retrieve_connection'
activerecord (6.1.4.1) lib/active_record/connection_handling.rb:283:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:611:in `connection'
activerecord (6.1.4.1) lib/active_record/migration.rb:606:in `build_watcher'
activerecord (6.1.4.1) lib/active_record/migration.rb:588:in `block in call'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `synchronize'
activerecord (6.1.4.1) lib/active_record/migration.rb:587:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
activesupport (6.1.4.1) lib/active_support/callbacks.rb:98:in `run_callbacks'
actionpack (6.1.4.1) lib/action_dispatch/middleware/callbacks.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/debug_exceptions.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
railties (6.1.4.1) lib/rails/rack/logger.rb:37:in `call_app'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `block in call'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `block in tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:37:in `tagged'
activesupport (6.1.4.1) lib/active_support/tagged_logging.rb:99:in `tagged'
railties (6.1.4.1) lib/rails/rack/logger.rb:26:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/request_id.rb:26:in `call'
rack (2.2.3) lib/rack/runtime.rb:22:in `call'
activesupport (6.1.4.1) lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/executor.rb:14:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/static.rb:24:in `call'
rack (2.2.3) lib/rack/sendfile.rb:110:in `call'
actionpack (6.1.4.1) lib/action_dispatch/middleware/host_authorization.rb:98:in `call'
railties (6.1.4.1) lib/rails/engine.rb:539:in `call'
puma (5.5.2) lib/puma/configuration.rb:249:in `call'
puma (5.5.2) lib/puma/request.rb:77:in `block in handle_request'
puma (5.5.2) lib/puma/thread_pool.rb:340:in `with_force_shutdown'
puma (5.5.2) lib/puma/request.rb:76:in `handle_request'
puma (5.5.2) lib/puma/server.rb:447:in `process_client'
puma (5.5.2) lib/puma/thread_pool.rb:147:in `block in spawn_thread'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:36:44 +0900
  
ActionController::RoutingError (No route matches [GET] "/api/v1/samples"):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:37:11 +0900
  
ActionController::RoutingError (uninitialized constant Api
Did you mean?  App):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:37:12 +0900
  
ActionController::RoutingError (uninitialized constant Api
Did you mean?  App):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:37:12 +0900
  
ActionController::RoutingError (uninitialized constant Api
Did you mean?  App):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:37:13 +0900
  
ActionController::RoutingError (uninitialized constant Api
Did you mean?  App):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:38:13 +0900
  
ActionController::RoutingError (uninitialized constant Api
Did you mean?  App):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:38:15 +0900
  
ActionController::RoutingError (uninitialized constant Api
Did you mean?  App):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:38:16 +0900
  
ActionController::RoutingError (uninitialized constant Api
Did you mean?  App):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 22:38:39 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 156)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 23:53:28 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 3ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 1978)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-27 23:53:29 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | ActiveRecord: 0.0ms | Allocations: 92)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 00:40:17 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 4ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 1978)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 00:40:42 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | ActiveRecord: 0.0ms | Allocations: 93)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 01:06:32 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 3ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 1979)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 01:06:34 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 92)


Started POST "/api/jsonws/invoke" for 45.146.164.110 at 2021-10-28 09:31:25 +0900
  
ActionController::RoutingError (No route matches [POST] "/api/jsonws/invoke"):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 10:23:48 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.6ms | Allocations: 138)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 10:23:48 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 11:57:02 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 22:10:28 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 4ms (Views: 0.6ms | ActiveRecord: 0.0ms | Allocations: 1978)


Started GET "/api/v1/samples" for 172.23.0.3 at 2021-10-28 22:10:39 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.3ms | ActiveRecord: 0.0ms | Allocations: 92)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 22:37:35 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 14ms (Views: 9.7ms | ActiveRecord: 0.0ms | Allocations: 1979)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 22:37:39 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.4ms | ActiveRecord: 0.0ms | Allocations: 93)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 23:32:18 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.3ms | Allocations: 93)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 23:32:19 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.3ms | Allocations: 92)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-28 23:43:59 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.4ms | Allocations: 92)


Started GET "/api/v1/samples" for 172.23.0.3 at 2021-10-28 23:51:18 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | Allocations: 88)


Started GET "/api/v1/samples" for 172.23.0.3 at 2021-10-28 23:51:19 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.2ms | Allocations: 88)


Started GET "/api/v1/samples" for 172.23.0.3 at 2021-10-28 23:52:24 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | Allocations: 88)


Started GET "/api/v1/notes" for 118.241.146.68 at 2021-10-29 00:14:17 +0900
Processing by Api::V1::NotesController#index as HTML
Completed 500 Internal Server Error in 15ms (ActiveRecord: 0.0ms | Allocations: 3608)


  
NameError (uninitialized constant Api::V1::NotesController::Note):
  
app/controllers/api/v1/notes_controller.rb:8:in `index'
Started GET "/api/v1/notes" for 118.241.146.68 at 2021-10-29 00:14:19 +0900
Processing by Api::V1::NotesController#index as HTML
Completed 500 Internal Server Error in 2ms (ActiveRecord: 0.0ms | Allocations: 1765)


  
NameError (uninitialized constant Api::V1::NotesController::Note):
  
app/controllers/api/v1/notes_controller.rb:8:in `index'
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:14:26 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 138)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:14:26 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 92)


Started GET "/api/v1/notes" for 118.241.146.68 at 2021-10-29 00:14:27 +0900
Processing by Api::V1::NotesController#index as HTML
Completed 500 Internal Server Error in 2ms (ActiveRecord: 0.0ms | Allocations: 1738)


  
NameError (uninitialized constant Api::V1::NotesController::Note):
  
app/controllers/api/v1/notes_controller.rb:8:in `index'
Started GET "/api/v1/sample" for 118.241.146.68 at 2021-10-29 00:14:40 +0900
  
ActionController::RoutingError (No route matches [GET] "/api/v1/sample"):
  
Started GET "/api/v1/sample" for 118.241.146.68 at 2021-10-29 00:14:41 +0900
  
ActionController::RoutingError (No route matches [GET] "/api/v1/sample"):
  
Started GET "/api/v1/sample" for 118.241.146.68 at 2021-10-29 00:14:42 +0900
  
ActionController::RoutingError (No route matches [GET] "/api/v1/sample"):
  
Started GET "/api/v1/sample" for 118.241.146.68 at 2021-10-29 00:14:52 +0900
  
ActionController::RoutingError (No route matches [GET] "/api/v1/sample"):
  
Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:14:54 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:14:54 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:15:09 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:15:15 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:15:18 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:15:19 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:16:16 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:16:20 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.1ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:16:21 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.3ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:16:21 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:16:30 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:16:33 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:16:35 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.3ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:17:44 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:18:00 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 89)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:18:02 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:18:03 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.3ms | ActiveRecord: 0.0ms | Allocations: 89)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:18:04 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:18:09 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:18:33 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.2ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:18:34 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.3ms | ActiveRecord: 0.0ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:22:26 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:22:43 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 1ms (Views: 0.1ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:22:46 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.2ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:22:48 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | Allocations: 88)


Started GET "/api/v1/samples" for 118.241.146.68 at 2021-10-29 00:22:49 +0900
Processing by Api::V1::SamplesController#index as HTML
Completed 200 OK in 0ms (Views: 0.1ms | Allocations: 88)


Started POST "/api/jsonws/invoke" for 45.146.164.110 at 2021-10-29 03:53:19 +0900
  
ActionController::RoutingError (No route matches [POST] "/api/jsonws/invoke"):
  
Started GET "/" for 35.80.12.9 at 2021-10-29 10:49:20 +0900
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.4.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.4.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 6.2ms | Allocations: 185)
Completed 200 OK in 18ms (Views: 8.0ms | Allocations: 1992)


Started GET "/system_api.php" for 134.209.18.129 at 2021-10-29 15:22:48 +0900
  
ActionController::RoutingError (No route matches [GET] "/system_api.php"):
  
Started GET "/c/version.js" for 134.209.18.129 at 2021-10-29 15:22:49 +0900
  
ActionController::RoutingError (No route matches [GET] "/c/version.js"):
  
Started GET "/streaming/clients_live.php" for 134.209.18.129 at 2021-10-29 15:22:50 +0900
  
ActionController::RoutingError (No route matches [GET] "/streaming/clients_live.php"):
  
Started GET "/stalker_portal/c/version.js" for 134.209.18.129 at 2021-10-29 15:22:51 +0900
  
ActionController::RoutingError (No route matches [GET] "/stalker_portal/c/version.js"):
  
Started GET "/stream/live.php" for 134.209.18.129 at 2021-10-29 15:22:52 +0900
  
ActionController::RoutingError (No route matches [GET] "/stream/live.php"):
  
Started GET "/flu/403.html" for 134.209.18.129 at 2021-10-29 15:22:53 +0900
  
ActionController::RoutingError (No route matches [GET] "/flu/403.html"):
  
Started POST "/api/jsonws/invoke" for 45.146.164.110 at 2021-10-29 23:43:57 +0900
  
ActionController::RoutingError (No route matches [POST] "/api/jsonws/invoke"):
  
Started GET "/" for 92.118.160.45 at 2021-10-30 08:40:43 +0900
Processing by Rails::WelcomeController#index as HTML
  Rendering /usr/local/bundle/gems/railties-6.1.4.1/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /usr/local/bundle/gems/railties-6.1.4.1/lib/rails/templates/rails/welcome/index.html.erb (Duration: 5.7ms | Allocations: 178)
Completed 200 OK in 7ms (Views: 6.6ms | Allocations: 714)


Started POST "/api/jsonws/invoke" for 45.146.164.110 at 2021-10-30 19:52:47 +0900
  
ActionController::RoutingError (No route matches [POST] "/api/jsonws/invoke"):
  
